tests/litmus-cleanup_test.go

```diff
@@ -111,32 +110,7 @@ var _ = Describe("BDD of litmus cleanup", func() {
 
 			fmt.Println("rbac deleted sucessfully")
 
-			//Deleting chaos-operator deployment
-			By("Deleting chaos-operator deployment")
-			err = exec.Command("kubectl", "delete", "deploy", "-n", "litmus", "--all").Run()
-			Expect(err).To(BeNil(), "failed to delete deployments")
-			if err != nil {
-				fmt.Println(err)
-			}
-
-			fmt.Println("chaos-operator deleted sucessfully")
-
-			//Checking Whether Litmus is Deleted successfully
-			operator, err := client.AppsV1().Deployments("litmus").Get("litmus", metav1.GetOptions{})
```

Comment:
Why are you removing this step?

Reply:
@uditgaurav Can you install and uninstall litmus via page repo link? cc: @ksatchit

Reply:
Beacuse when rbac will be deleted then litmus namespace will be deleted which will be end of story.

Reply:
If I use pages repo then I have to install/remove each of the components separately which will add extra task ... so I is good to install/remove from chaos-operator.

Reply:
@uditgaurav If you use page repo link then you can install/remove whole litmus in one task only. It may reduce the no. of tasks.